metricbeat/module/docker/memory/memory.go

```diff
@@ -41,25 +45,28 @@ type MetricSet struct {
 	memoryService *MemoryService
 	dockerClient  *client.Client
 	dedot         bool
+	logger        *logp.Logger
 }
 
 // New creates a new instance of the docker memory MetricSet.
 func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
+	logger := logp.NewLogger(metricsetName)
```

Comment:
Can we create the logger with name "docker.memory" instead of just "memory", to deal with the ambiguity you point out?

Reply:
Logger name update to "docker.memory".